packages/terra-avatar/src/common/AvatarUtils.jsx

```diff
@@ -60,17 +60,21 @@ const getColorVariant = (hashValue) => {
 /**
  * Render placeholder.
  */
-const generateImagePlaceholder = (alt, isAriaHidden, variant) => {
-  const avatarIconClassNames = cx(['icon', variant]);
+const generateImagePlaceholder = (alt, isAriaHidden, variant, initials) => {
+  if (initials && initials.length <= 2) {
+    const avatarTextClassNames = cx('initials');
+    return <span className={avatarTextClassNames} alt={alt} aria-label={alt} aria-hidden={isAriaHidden}>{initials.toUpperCase()}</span>;
+  }
 
+  const avatarIconClassNames = cx(['icon', variant]);
```

Comment:
Based on new UX requirements, we should remove this icon logic and default the placeholder to be initials.

Reply:
I was bit confused with new Ux requirements for removing fallback icon from avatar. since there   was no response on the comment  I made in [Issue page](https://github.com/cerner/terra-core/issues/2559#issuecomment-526664922) I Proceeded with logic which adds fallback to initials without changing the existing default Icon fallback logic.

Reply:
Let's sync with UX on this

cc @neilpfeiffer @jeremyfuksa

Reply:
Got approval from @jeremyfuksa for having fallback only when initials are provided : [conversation-link](https://github.com/cerner/terra-core/issues/2559#issuecomment-529600379)